lib/engine/config/game/g_18_cz.rb

```diff
@@ -303,126 +303,576 @@ module G18CZ
+                               "8866p",
+                               "216p",
+                               "611p",
+                               "889p",
+                               "8894p",
+                               "8895p",
+                               "8896p",
+                               "595p",
+                               "8857p"
+                            ],
+                            "when": "track",
+                            "hexes": [],
+                            "reachable": true
+                        }
+                    ]
                 },
                 {
                     "name": "Wekelsdorf – Parschnitz – Trautenau",
                     "value": 70,
                     "revenue": 20,
                     "sym": "WPT",
-                    "abilities": []
+                    "abilities": [
+                        {
+                            "type":"tile_lay",
+                            "count": 1,
+                            "owner_type":"corporation",
+                            "free": false,
+                            "tiles":[
+                               "14p",
+                               "15p",
+                               "887p",
+                               "888p",
+                               "8866p",
+                               "216p",
+                               "611p",
+                               "889p",
+                               "8894p",
+                               "8895p",
+                               "8896p",
+                               "595p",
+                               "8857p"
+                            ],
+                            "when": "track",
+                            "hexes": [],
+                            "reachable": true
+                        }
+                    ]
                 },
                 {
                     "name": "Nezamislitz – Morkowitz",
                     "value": 75,
                     "revenue": 20,
                     "sym": "NM",
-                    "abilities": []
+                    "abilities": [
+                        {
+                            "type":"tile_lay",
+                            "count": 1,
+                            "owner_type":"corporation",
+                            "free": false,
+                            "tiles":[
+                               "14p",
+                               "15p",
+                               "887p",
+                               "888p",
+                               "8866p",
+                               "216p",
+                               "611p",
+                               "889p",
+                               "8894p",
+                               "8895p",
+                               "8896p",
+                               "595p",
+                               "8857p"
+                            ],
+                            "when": "track",
```

Comment:
I think `when` s.b . "owning_corp_or_turn" since it can be laid at any point in the corp's turn

Reply:
yeah, you're right. Fixed it